src/Emulator/Visuals/Visuals.tsx

```diff
@@ -98,7 +123,7 @@ const Visuals = () => {
         onStop={() => setIsGrabbing(false)}
         onDrag={handleDrag}
       >
-        <div className="handle" style={{ cursor: isGrabbing ? '-webkit-grabbing': '', visibility: hideImage ? 'hidden' : 'initial' }}>
+        <div className="handle" style={{ cursor: isGrabbing ? '-webkit-grabbing': '', visibility: hideImage ? 'hidden' : 'initial', backgroundColor: '#212529', borderRadius: '10px' }}>
```

Comment:
I would try to break up this element into multiple lines as well because the `style` object is a little large and has some logic in it.
```jsx
<div
  className="handle"
  style={{
    cursor: isGrabbing ? '-webkit-grabbing': '',
    visibility: hideImage ? 'hidden' : 'initial',
    backgroundColor: '#212529',
    borderRadius: '10px' 
  }}
>
```